pkg/config/polling_manager.go

```diff
@@ -79,6 +78,13 @@ func Requester(requester utils.Requester) OptionFunc {
 	}
 }
 
+// DatafileTemplate is an optional function, sets a passed datafile URL template
```

Comment:
`DatafileURLTemplate`

Reply:
const DatafileURLTemplate = "https://cdn.optimizely.com/datafiles/%s.json", go does not allow to use the same name for a var and a function